client/me/purchases/cancel-purchase/button.jsx

```diff
@@ -7,17 +7,27 @@ import React from 'react';
 /**
  * Internal Dependencies
  */
+import wpcom from 'lib/wp';
```

Comment:
We should never import `wpcom` lib from UI components. Have you looked at dispatching the actions with redux instead?